packages/web-app-files/src/components/FileLinkSidebar.vue

```diff
@@ -14,13 +14,7 @@
             Any external person with the respective link can access this resource. No sign-in
             required. Assign a password to avoid unintended document exposure.
           </p>
-          <oc-button
-            id="files-file-link-add"
-            icon="add"
-            variation="primary"
-            :aria-label="$_addButtonAriaLabel"
-            @click="addNewLink"
-          >
+          <oc-button id="files-file-link-add" icon="add" variation="primary" @click="addNewLink">
```

Comment:
I didn't understand the ticket in a way that removing the `:aria-label` already satisfies the requirements